### DISCUSSION TITLE
Date

### DISCUSSION THREAD

**Discussion by jabamaus** [mruby/mruby#6100] [General]
Hi. Is the Date class available in mruby or as a gem? I can't find it...

**Comment by matz** [comment]
We usually recommend to use `Time` class instead. Do you have any specific need for `Date` class?

**Comment by jabamaus** [comment]
My codebase is attempting to be 'bilingual' and work fully with mruby and cruby (mostly for the ruby debug gem which I wish worked with mruby - separate discussion). My codebase uses Date currently but I will look at moving to Time which should be fine.

**Comment by matz** [comment]
Unless you are working with far distant time (post 2039 or pre 1903 on 32 bit platforms), `Time` should be fine.
If I have spare time (unfortunately I rarely have them), I would work on mruby `Date` gem, just maybe.

**Comment by jabamaus** [comment]
That's fine. Don't worry Matz, I'm sure there are more important things to do :)
